src/Symfony/Component/Console/Attribute/Argument.php

```diff
@@ -134,4 +134,12 @@ public function getInteractiveAttribute(): ?InteractiveAttributeInterface
     {
         return $this->interactiveAttribute;
     }
+
+    /**
+     * @internal
```

Comment:
can we do without the internal-under-the-hood API?
`->toInputArgument()->isRequired()`?
but I'm also seeing that toInputArgument is internal, and a few others.
We should have less internal methods IMHO. Internal means nobody else but us can wire stuff together. That feels odd for a reusable piece of software.

Reply:
In this particular case, I wanted to save some ticks/mem (creating a full InputArgument object just to ask for `isRequired()`) when this information is just right here in this class